config/tidb.toml

```diff
@@ -43,7 +43,7 @@ enable-table-lock = true
 # alter-primary-key is used to control alter primary key feature. Default is false, indicate the alter primary key feature is disabled.
 # If it is true, we can add the primary key by "alter table", but we may not be able to drop the primary key.
 # In order to support "drop primary key" operation , this flag must be true and the table does not have the pkIsHandle flag.
-alter-primary-key = true
```

Comment:
you need to check if it is any tests depend on this config being trueyou need to check if it is any tests depend on this config being true